eo-parser/src/main/java/org/eolang/parser/ParsingTrain.java

```diff
@@ -89,6 +84,7 @@ public final class ParsingTrain extends TrEnvelope {
         "/org/eolang/parser/warnings/duplicate-metas.xsl",
         "/org/eolang/parser/warnings/mandatory-package-meta.xsl",
         "/org/eolang/parser/warnings/mandatory-home-meta.xsl",
+        "/org/eolang/parser/warnings/mandatory-version-meta.xsl",
```

Comment:
We need to add `incorrect-version.xsl` to this pipeline too, like `incorrect-home` and `incorrect-architect.xsl`